mailmerge/sendmail_client.py

```diff
@@ -124,9 +124,19 @@ def sendmail_ssltls(self, sender, recipients, message):
         except ssl.SSLError as err:
             raise exceptions.MailmergeError(f"SSL Error: {err}")
         host, port = (self.config.host, self.config.port)
+
+        def needs_smtputf8(*string):
+            return any(any(ord(c) > 127 for c in part) for part in string)
```

Comment:
Leave a note in a comment that this can be improve if/when we move to Python 3.7.  Currently mailmerge supports Python 3.6+

https://stackoverflow.com/questions/196345/how-to-check-if-a-string-in-python-is-in-ascii